### PR TITLE
Recording resource code samples

### PR DESCRIPTION
**Pull request by sarahcstringer**
Add new code samples for the room Recording Resource.

### CODE REVIEW

File: video/rest/rooms/update-recording-rules-only-record-audio-all/update-recording-rules-only-record-audio-all.7.x.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  public static final String ACCOUNT_SID = "ACXXXX";
+  public static final String API_KEY_SID = "SKXXXX";
+  public static final String API_KEY_SECRET = "your_api_key_secret";
+
+  public static void main( String[] args ) throws IOException{
+      // Initialize the client
+      Twilio.init(API_KEY_SID, API_KEY_SECRET, ACCOUNT_SID);
+      
+      ArrayList<RecordingRule> rules = new ArrayList<>();
+	  rules.add(RecordingRule.builder()
```
Comment (craigsdennis) [inline video/rest/rooms/update-recording-rules-only-record-audio-all/update-recording-rules-only-record-audio-all.7.x.java:24]: alignment here seems off

File: video/rest/rooms/update-recording-rules-stop-recording-all-participants/update-recording-rules-stop-recording-all-participants.7.x.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class UpdateRules{
+
+  // Find your credentials at twilio.com/console
+  public static final String ACCOUNT_SID = "ACXXXX";
+  public static final String API_KEY_SID = "SKXXXX";
+  public static final String API_KEY_SECRET = "your_api_key_secret";
+
+  public static void main( String[] args ) throws IOException{
+      // Initialize the client
+      Twilio.init(API_KEY_SID, API_KEY_SECRET, ACCOUNT_SID);
```
Comment (craigsdennis) [inline video/rest/rooms/update-recording-rules-stop-recording-all-participants/update-recording-rules-stop-recording-all-participants.7.x.java:21]: Should we use Account SID and Auth Token?

Comment (craigsdennis) [reply video/rest/rooms/update-recording-rules-stop-recording-all-participants/update-recording-rules-stop-recording-all-participants.7.x.java:21]: Or update the note on where to find them

**Review by craigsdennis — APPROVED** [review]
LGTM with a few tweaks

File: video/rest/rooms/update-recording-rules-stop-recording-all-participants/update-recording-rules-stop-recording-all-participants.7.x.java (hunk 2 above)
Comment (sarahcstringer) [reply video/rest/rooms/update-recording-rules-stop-recording-all-participants/update-recording-rules-stop-recording-all-participants.7.x.java]: 👍 switched to using Account SID and Auth Token

File: video/rest/rooms/update-recording-rules-only-record-audio-all/update-recording-rules-only-record-audio-all.7.x.java (hunk 1 above)
Comment (sarahcstringer) [reply video/rest/rooms/update-recording-rules-only-record-audio-all/update-recording-rules-only-record-audio-all.7.x.java]: Thanks for catching! Copy paste weirdness.

**Pull request merged**